NOpenCL/CommandQueue.cs

```diff
@@ -8,7 +8,10 @@ namespace NOpenCL
     using System;
     using System.ComponentModel;
     using NOpenCL.SafeHandles;
+    using System.Diagnostics;
+    using System.Runtime.InteropServices;
 
+    [DebuggerDisplay("Device = {Device.Name}")]
```

Comment:
I see, that makes sense.  All changes are approved in one large chunk so small parts cannot be approved. Also, if there are several outstanding pull requests at the same time then they would be more difficult to merge. Thank you for your help.